src/System.Text.Json/src/System/Text/Json/Node/JsonString.cs

```diff
@@ -82,7 +82,7 @@ public string Value
         /// <exception cref="FormatException">
         ///   Text value of this instance is not in an ISO 8601 defined DateTime format.
         /// </exception>
-        public DateTime GetDateTime() => DateTime.ParseExact(_value, "s", CultureInfo.InvariantCulture);
+        public DateTime GetDateTime() => JsonHelpers.TryParseAsISO(_value.AsSpan(), out DateTime value) ? value : throw new FormatException(SR.FormatDateTime);
```

Comment:
Ditto, but for parsing

Reply:
Same as above. Ideally, this would just use `DateTime.Parse(_value, "O", Culture.InvariantCulture)`, but that format is a bit too strict compared to what ISO allows. That's why `JsonHelpers.TryParseAsISO` exist in the first place.